crates/webidl/src/first_pass.rs

```diff
@@ -292,12 +339,55 @@ impl<'src> FirstPass<'src, ()> for weedle::PartialInterfaceMixinDefinition<'src>
         record
             .mixins
             .entry(self.identifier.0)
-            .or_insert_with(Default::default)
+            .or_insert_with(||
+                MixinData {
```

Comment:
It is because of `partial: true`.

Reply:
Yeah, so we couldn't `derive(Default)` for `MixinData`, but we could do

```rust
impl Default for MixingData {
    fn default() -> MixinData {
        MixinData {
            partial: true,
            ...
    }
}
```

Reply:
That probably wound not be useful because `partial: false` is also used: https://github.com/rustwasm/wasm-bindgen/pull/679/files#diff-73533b774757f58254639c3bc83e14d8R320